pkg/cmd/server/server.go

```diff
@@ -565,6 +565,7 @@ func (s *server) runControllers(config *api.Config, defaultBackupLocation *api.B
 	}
 
 	backupSyncController := controller.NewBackupSyncController(
+		s.arkClient.ArkV1(),
```

Comment:
Is there a reason we shouldn't use the same object in memory for both of these?

Reply:
Eh, this is how we do it everywhere, though I find this pattern annoying.  I'd rather switch controllers to either (a) just take an `ArkV1Interface` and let them access whichever type-specific getters they need, or (b) take namespace-specific clients (e.g. `s.arkClient.ArkV1().Backups(s.namespace)`)

Reply:
Yeah, I think if we're dealing with multiple Ark resources in a single place, these would make sense. My initial suggestion was going to be (a).

Reply:
Pushed a commit showing what (b) would look like

Reply:
@nrb if it's OK with you I'd rather tackle this problem as part of a different PR, and use the existing pattern here for now.

Reply:
Yep, that's fine.